scripts/publish.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+export TEST_SUITE_TYPE="build"
+
+if [ "$BRANCH" == "master" ];
+then
+  echo "attempting npm publish"
+
+  export REGISTRY="${ARTIFACTORY_URL}/api/npm/npm-okta"
+  node $OKTA_HOME/$REPO/scripts/publish_registry.js
```

Comment:
given there is only one module, I doubt you need the `publish_registry.js` but only `npm publish --registry ${registry}`?